### ISSUE TITLE
False positive for W0602

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#195]
Originally reported by: **the mulhern (BitBucket: [the_mulhern](http://bitbucket.org/the_mulhern))**

---

Mini-example:

```
#!python

path = None

def junk():
    global path
    from os import path

def junk2():
    print("%s" % path)
```

Pylint reports W0602 (global-variable-not-assigned) for the statement "global path".

But actually, path is assigned to by means of the import statement as can be seen by the interpreter interaction below:

> > > from junk import *
> > > junk2()
> > > None
> > > junk()
> > > junk2()
> > > <module 'posixpath' from '/usr/lib64/python2.7/posixpath.pyc'>

Without the global statement the second invocation of junk2() would print None, just like the first.

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/195
